config/samples/spec_v1alpha3_clusterworkloadresourcemapping.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: servicebinding.io/v1alpha3
```

Comment:
```suggestion
apiVersion: servicebinding.io/v1beta1
```

Reply:
This is meant for the older `v1alpha3` version (see the file name). `v1beta1` is covered by the `spec_v1beta1_clusterworkloadresourcemapping.yaml`

Reply:
You see, when operator-sdk validates bundle, it complains if an example is missing for any CRD version owned by CSV. Which in our case includes both `v1alpha3` and `v1beta2` versions of the SPEC APIs.

Reply:
That is why I included all of the missing ones.